packages/ui/src/components/breath-player/animation-steps.js

```diff
@@ -0,0 +1,43 @@
+const stampAnimation = () => ({
+  scale: [],
+  duration: [],
+  outerScale: [],
+})
+
+const createNArray = (n, fn) => new Array(n).fill(0).map((_, i) => fn(i))
+
+export const defaults = {
+  // circle scales
+  scale: 1,
+  scaleExpanded: 1.6,
+  outerScale: 1,
+  outerExpanded: 2,
+  scaleExpandedIncrease: 0.7,
+  outerExpandedIncrease: 1,
+  // duration
+  breathTime: 3000,
+  breathTimeIncrease: 1500,
+}
+
+export const createAnimationSteps = amount => {
```

Comment:
I can also see the ability to create multiple functions here to support different breathing exercises.